HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,13 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: isLocal
+Returns true if current server is local
```

Comment:
Could you please add an explanation, here is my suggestion:

Returns true if the server (scsynth) runs on the same computer as  the language (sclang).